tests/v2/validation/rbac/rbac.go

```diff
@@ -35,6 +36,7 @@ const (
 	psaWarn             = "pod-security.kubernetes.io/warn"
 	psaAudit            = "pod-security.kubernetes.io/audit"
 	psaEnforce          = "pod-security.kubernetes.io/enforce"
+	resAdminSetting     = "kubeconfig-token-ttl-minutes"
```

Comment:
I think this can be called something related to setting the name instead of the client that triggers it.